nautobot_secrets_providers/providers/hashicorp.py

```diff
@@ -151,11 +151,14 @@ def get_client(cls, secret=None):
                 elif auth_method == "aws":
                     session = boto3.Session()
                     aws_creds = session.get_credentials()
+                    aws_region = session.region_name
+                    if aws_region is None:
+                        aws_region = "us-east-1"
```

Comment:
Nit pick pls do not hate me.
```suggestion
                    aws_region = getattr(session, "region_name", "us-east-1")
```

Reply:
That's functionally different though - `session.region_name` is `None`, not nonexistent.

Reply:
Could we do this then?
```suggestion
                    aws_region = session.region_name or "us-east-1"
```

Reply:
@glennmatthews Thoughts on the above or is the current good enough?

Reply:
Either approach is fine by me.